### PR TITLE
emacsclient: add page

### PR DESCRIPTION
**Pull request by jaseemabid**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/emacsclient.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# emacsclient
+
+> Open files in an existing emacs server.
+
+- Open file in console mode (without X window):
+
+`emacsclient -nw {{filename}}`
+
+- Open a file and not wait for it:
```
Comment (agnivade) [inline pages/common/emacsclient.md]: not wait for it _to do what ?_

**Comment by agnivade** [comment]
Thanks. One last question though - does this work in Windows ? If not, we should have separate entries for linux and mac.

**Comment by jaseemabid** [comment]
@agnivade I've no idea. Happy to move it to linux category.

**Comment by agnivade** [comment]
Seems to be there - https://www.emacswiki.org/emacs/EmacsMsWindowsIntegration

File: pages/common/emacsclient.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# emacsclient
+
+> Open files in an existing emacs server.
+
+- Open file in console mode (without X window):
```
Comment (sbrl) [inline pages/common/emacsclient.md]: You mention console mode here, but hint at a graphical interface. Perhaps an example demonstrating the graphical interface might be worth including here?

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @jaseemabid :D

**Pull request merged**